project/src/main/webapp/events-feed.html

```diff
@@ -63,7 +64,8 @@ <h3>Discover Events Around You</h3>
     </div>
     <br />
     <br />
-    <div class="container" id="event-container">
+    <div class="grid" id="event-container">
+      <div class="grid-sizer"></div>
```

Comment:
can we max grid size to like 4 or 5 events per row and not dynamically resize the width? like it'll always have space for x events but not change size if there are eg 3 events

Reply:
Good point - the width is actually not dynamically resized, and if you don't fill up a column the size stays the same (and it wraps to the next row if it's too many events)